src/index.js

```diff
@@ -0,0 +1,87 @@
+import './style.css';
+import DotsIcon from './more_vert_black_24dp.svg';
+import EnterIcon from './keyboard_return_black_24dp.svg';
+import RefershIcon from './replay_black_24dp.svg';
+
+const toDoList = [
+  {
+    description: 'Do Workout',
+    completed: false,
+    number: '1',
+  },
+  {
+    description: 'Clean my room',
+    completed: false,
+    number: '2',
+  },
+  {
+    description: 'Take my breakfast',
+    completed: false,
+    number: '3',
+  },
+  {
+    description: 'Take my shower',
+    completed: false,
+    number: '4',
+  },
+  {
+    description: 'Complete all the Microverse day activities',
+    completed: false,
+    number: '5',
+  },
+];
```

Comment:
- [ ] **Required:** You have done an amazing job so far. 👏 However, the is one thing you have to do to move to the next milestone.
The items are required to display in the order of the `index`. Assuming the index are unordered received as the worst case.

_**Hint**_ Sort the items by index before render. `Array.sort((a, b) => a.index - b.index);`